### PR TITLE
Gutenboarding import flow: additonal adjustments

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

Changes we are introducing in this PR are:
- add alternative intent links group (import your site content)
- enable initial back button
- add proper learn more link pointing to the official documentation
- add valid links to the multiple importers we are supporting on the import list page

#### Testing instructions

**NOTE:** Query params: flags and siteSlug are required.

* Go to: `/start/setup-site/intent?flags=signup/hero-flow&siteSlug={YOUR_SITE_SLUG}`
* Click on the `Import your site content` button
* Click on the `Back` button

--

* Go to Importers List page by clicking on `I don't have a site address` button
* Clicking on the particular importer, you will be redirected to the Ready screen

--

* Go to Ready page
* Click on the `What can be imported?` button
* Click on the `Learn more` button

#### Screenshots
Screenshot 2021-11-01 at 18 46 31

Screenshot 2021-11-01 at 18 57 13

Related to:
- [Figma Design](https://www.figma.com/file/TAK5FpwTvfbENJDfHc7Aq1/Site-Importing?node-id=1930%3A21706)

### CODE REVIEW

File: client/signup/steps/import/ready/platform-details.tsx [hunk 1]
```diff
@@ -52,7 +53,7 @@ const ImportPlatformDetails: React.FunctionComponent< DetailsProps > = ( data )
 				);
 			case 'Medium':
 				return __(
-					"Our Mediun content importer is the quickest way to move your content. Simply export the contents from Medium as a .ZIP file, then click 'Import your content' and upload it to our importer."
+					"Our Medium content importer is the quickest way to move your content. Simply export the contents from Medium as a .ZIP file, then click 'Import your content' and upload it to our importer."
```
Comment (a8ci18n) [inline client/signup/steps/import/ready/platform-details.tsx:56]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by vishnugopal** [comment]
Some changes I could see in the flow:

Make the entire bar clickable instead of just the right arrow:
CleanShot 2021-11-02 at 09 51 10@2x

Clicking on the right arrow / bar should lead to the Import No URL page, not direct to the importer:
CleanShot 2021-11-02 at 09 51 46@2x

Otherwise things looks fine!

**Review by vishnugopal — CHANGES_REQUESTED** [review]
Just make the changes above, and things should be good to go!

**Comment by bogiii** [comment]
> Make the entire bar clickable instead of just the right arrow:

For listing available importers, we are consuming **ActionCard** component. There is no option to make the whole action card clickable.

Integration piece of code looks:
```
<ActionCard
	classNames={ 'list__importer-action' }
	headerText={ 'Blogger' }
	mainText={ 'www.blogger.com' }
	buttonIcon={ 'chevron-right' }
	buttonHref={ getWpComImporterUrl( siteSlug, 'blogger' ) }
/>
```

A potential solution, to position the button in absolute mode and stretch it to the full bar width?
^ @vishnugopal

**Comment by bogiii** [comment]
> Just make the changes above, and things should be good to go!

The requested changes have been made, so I'm good to go.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6942420

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

File: client/signup/steps/intent/index.tsx [hunk 2]
```diff
@@ -30,10 +36,15 @@ export default function IntentStep( props: Props ): React.ReactNode {
 	const branchSteps = useBranchSteps( stepName );
 
 	const submitIntent = ( intent: IntentFlag ) => {
-		branchSteps( EXCLUDE_STEPS[ intent ] );
-		recordTracksEvent( 'calypso_signup_intent_select', { intent } );
-		dispatch( submitSignupStep( { stepName }, { intent } ) );
-		goToNextStep();
+		recordTracksEvent( 'calypso_signup_select_intent', { intent } );
```
Comment (arthur791004) [inline client/signup/steps/intent/index.tsx:39]: Hmmm...I just found that the event name changed back to the one before renaming...

Comment (arthur791004) [inline client/signup/steps/intent/index.tsx:39]: Hmmm...I just found that the event name changed back to the one before renaming...

Fix in https://github.com/Automattic/wp-calypso/pull/57887

Comment (bogiii) [reply client/signup/steps/intent/index.tsx:39]: I'm not sure what is happening here; maybe it was related to "rebase conflict resolution".
Sorry for that and thanks for letting me know.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6942420).

**Pull request merged**